tests/conftest.py

```diff
@@ -136,8 +137,19 @@ def _make_client(auth: Tuple[str, str]) -> AsyncClient:
 
 
 @pytest.fixture(autouse=True)
-def flush_server(server: str):
-    assert requests.post(f"{server}/__flush__")
+async def flush_default_collection(
+    make_client: Callable[[Tuple[str, str]], AsyncClient],
+    auth: Tuple[str, str],
+    source_bucket: str,
+    source_collection: str,
+):
+    client = make_client(auth)
```

Comment:
```suggestion
    yield
    client = make_client(auth)
```

I don't know how important this actually is, but since this feels like a cleanup step for tests, I think we want to put `client.delete_collection` after a yield statement. See [this section](https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended) of the docs.

Reply:
Good call! I like this a lot, thanks!

Reply:
So if I understand correctly, without the `yield` the flush happens before the test execution.

We may want to flush before, to make sure we start with the right context? (or both if we want to leave the server clean)

Reply:
@leplatrem I agree with starting off with the right context, but my thinking here is that we should clean up _after_ each test to ensure this, rather than tearing things down in the "set up" phase of tests.